x-pack/plugins/features/server/feature_schema.ts

```diff
@@ -155,6 +170,11 @@ const kibanaIndependentSubFeaturePrivilegeSchema = schema.object({
     read: schema.arrayOf(schema.string()),
   }),
   ui: listOfCapabilitiesSchema,
+  replacedBy: schema.maybe(
```

Comment:
Question: why for subfeatures we don't allow `minimal`/`default`?

Reply:
We’ve already discussed this offline, but just for the record: minimal privileges only exist for top-level privileges, as "minimal" means a privilege without any associated sub-feature privileges. Currently, sub-feature privileges cannot include or be associated with other sub-feature privileges. We use "minimal" privileges under the hood when admins customize sub-feature privileges for a given top-level privilege.

For example, let’s assume we have a feature Alpha with `All` and `Read` top-level privileges, and `Sub-alpha-1` and `Sub-alpha-2` sub-feature privileges, which are b**y default included** in the `All` top-level privilege. When an admin toggles the `All` privilege for feature Alpha and doesn’t change anything else, the resulting role will only have the `feature-alpha.all` privilege, which assumes/includes both `sub-alpha-1` and `sub-alpha-2`. However, if the admin decides to customize sub-feature privileges and toggles off `Sub-alpha-2`, the resulting role will include `feature-alpha.minimal_all` and `feature-alpha.sub-alpha-1` thus excluding `feature-alpha.sub-alpha-2` that's included in `feature-alpha.all`, but not in `feature-alpha.minimal_all`.

I'll add explanation to `x-pack/packages/security/authorization_core_common/src/privileges/minimal_privileges.ts`

Reply:
Added in https://github.com/elastic/kibana/pull/186800/files#diff-2cd66c055c746debabf96acb085cd199dde7cda22a7d5639b03cf7b696fa8d80R8-R24